device-registry/src/main/resources/db/migration/V5__system_info.sql

```diff
@@ -0,0 +1,7 @@
+
+CREATE TABLE DeviceSystem (
+  uuid char(36) NOT NULL,
+  system_info LONGTEXT NOT NULL,
```

Comment:
Yeah I agree it should be json but I was looking for something like postgres json type. We could then run operations on specific fields of that json object and even create indexes on it, but it is only available for mariadb 10.2, so this will have to do for now.